### PR TITLE
authz: various fixes to `PermsSyncer`

### PR DESCRIPTION
**Pull request by unknwon**
This PR:
1. Fixes SQL error with raw time string.
2. Adds logging upon completing repo and user perms sync.
3. Change schedule to 1 minute to be more responsive to users and repos with no permissions.

NOTE: Tests for `PermsSyncer` are on the way but not included in this PR.

### CODE REVIEW

File: enterprise/cmd/repo-updater/authz/perms_syncer.go [hunk 1]
```diff
@@ -569,8 +571,8 @@ func (s *PermsSyncer) schedule(ctx context.Context) (*schedule, error) {
 	//   initial limit  = <predicted from the previous step>
 	//	 consumed by users = <initial limit> / (<total repos> / <page size>)
 	//   consumed by repos = (<initial limit> - <consumed by users>) / (<total users> / <page size>)
-	// Hard coded both to 100 for now.
-	const limit = 100
+	// Hard coded both to 10 for now.
+	const limit = 10
```
Comment (tsenart) [inline enterprise/cmd/repo-updater/authz/perms_syncer.go]: @unknwon: For the initial syncing of permissions, on a fresh Sourcegraph instance, updating 10 repos / users at time, with a 1 minute interval, would likely take far too long? What do you think?

Comment (unknwon) [reply enterprise/cmd/repo-updater/authz/perms_syncer.go]: Oh, this limit is for getting rows with stale permissions, for repos and users with no permissions, we always list and enqueue all of them. Does this sound right?

Comment (tsenart) [reply enterprise/cmd/repo-updater/authz/perms_syncer.go]: > we always list and enqueue all of them. Does this sound right?

Yes, it does. Can you point me to the code that lists and enqueues all of them?

Comment (unknwon) [reply enterprise/cmd/repo-updater/authz/perms_syncer.go]: They are actually just few lines above:

https://github.com/sourcegraph/sourcegraph/blob/4cef40f321d3bce5f0cae18ba942e326c51883c4/enterprise/cmd/repo-updater/authz/perms_syncer.go#L554-L564

**Pull request merged**